lib/backends/wasm/ops/clip.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {Clip} from '../../../ops/clip';
+import {Tensor} from '../../../tensor';
+import {WasmBinding} from '../../../wasm-binding';
+import {WasmInferenceHandler} from '../inference-handler';
+
+export class WasmClip extends Clip {
+  run(inferenceHandler: WasmInferenceHandler, inputs: Tensor[]): Tensor[] {
+    const result = new Tensor(inputs[0].dims, inputs[0].type);
+    const size = result.floatData.length;
+    if (inputs[0].type === 'float32') {
+      WasmBinding.getInstance().ccall(
+          '_clip_f32', [inputs[0].floatData, 'float32ptr'], [result.floatData, 'float32ptr', 'out'], [size, 'int32'],
```

Comment:
No use of templatizing as `unary-op` in WASM backend as there is not much to templatize (no core implementation here). In fact, the overhead in dealing with types and attributes parsing (which is much more prevalent than in `binary-op` case makes it an unattractive option. So, keep the kernels separate even for `unary-op` like implementations